python/ray/tests/BUILD

```diff
@@ -316,7 +316,7 @@ py_test_module_list(
     "test_multiprocessing.py",
     "test_list_actors.py",
   ],
-  size = "medium",
+  size = "large",
```

Comment:
Did this test start taking longer because of this PR?

Reply:
There seems to be an issue with test_basic_2.py test_actor_recursive, but I'm not able to reproduce it locally. Still investigating.